app/assets/javascripts/simple-data-grapher.js

```diff
@@ -0,0 +1,73 @@
+function determineType2(graphType){
+    if (graphType === "Horizontal" || graphType === "Vertical"){
+        return "bar";
+    } 
+    else if (graphType === "Pie" || graphType === "Doughnut" || graphType === "Radar" ){
+        return "pie";
+    }
+    else {
+        return "scatter";
+    }
+}
+function layoutMaker(graphType){
+    let layout = {};
+    if (graphType === "Horizontal" || graphType === "Vertical"){
+        layout["barmode"] = "group";
+    }
+    return layout;
+}
+function traceMaker(graphType){
+    let trace = {};
+    trace["type"] = determineType2(graphType);
+    if (graphType === "Horizontal"){
+        trace["orientation"] = "h";
+    }
+    else if (graphType === "Doughnut"){
+        trace["hole"] = 0.5;
+    }
+    else if (graphType === "Basic"){
+        trace["mode"] = "lines";
+    }
+    else if (graphType === "Point"){
+        trace["mode"] = "markers";
+    }
+    else if (graphType === "Stepped"){
+        trace["mode"] = "lines+markers";
+        trace["line"] = {"shape": 'hv'};
```

Comment:
whitespace after `{` and before `}`.

Reply:
> @IshaGupta18 Left some comments.
> Also as a side task, I suggest integrating a linter in your editor and reading more about linting rules (eslint) for js. It will help you to get accommodated to guidelines.

I already have that in my editor (vs code), it's just not working for this folder, otherwise it works fine!